spec/requests/complete_booking_request_spec.rb

```diff
@@ -21,6 +21,7 @@
         }
       }
 
+      allow(BookingLocations).to receive(:find).and_return(double(:booking_location, name_for: ''))
```

Comment:
Extracted this out to a before with a comment on why it's needed. Let me know if it makes more sense now.